src/neuroconv/datainterfaces/ecephys/spike2/requirements.txt

```diff
@@ -1 +1 @@
-sonpy>=1.7.1;python_version<='3.9'
+sonpy>=1.7.1;python_version=='3.9'
```

Comment:
@bendichter I don't think I have Will Collins's contact info - can you remind them of the need to release `sonpy` for Python >= `3.10`?